tests/integration_tests/test_examples.py

```diff
@@ -151,3 +152,36 @@ def test_pearl_ml1_push():
         '--embedding_mini_batch_size', '2', '--max_path_length', '1'
     ],
                           check=False).returncode == 0
+
+
+@pytest.mark.no_cover
+@pytest.mark.timeout(400)
+def test_rl2_ppo_halfcheetah():
+    """Test rl2_ppo_halfcheetah.py."""
+    assert subprocess.run([
+        EXAMPLES_ROOT_DIR / 'tf/rl2_ppo_halfcheetah.py', '--n_epochs', '1',
+        '--episode_per_task', '1', '--meta_batch_size', '1'
```

Comment:
`wrap_experiment` does not handle console arguments automatically. You need to use `click`.

Reply:
fixed